docs/content/preview/architecture/transactions/concurrency-control.md

```diff
@@ -0,0 +1,1164 @@
+---
+title: Concurrency control
+headerTitle: Concurrency control
+linkTitle: Concurrency control
+description: Details about Concurrency control in YSQL
+menu:
+  preview:
+    identifier: architecture-concurrency-control
+    parent: architecture-acid-transactions
+    weight: 1153
+type: docs
+---
+
+[Concurrency control](https://en.wikipedia.org/wiki/Concurrency_control) in databases ensures that multiple transactions can execute concurrently while preserving data integrity. Concurrency control is essential for correctness in environments where two or more transactions can access the same data at the same time.
+
+YugabyteDB provides 2 policies to handle conflicts between concurrent transactions as described below. Also, see section at the end for information on how row-level explicit locking clauses behave with these concurrency control policies.
+
+## Fail-on-Conflict
+
+This is the default concurrency control strategy and is applicable for `REPEATABLE READ` and `SERIALIZABLE` isolation levels.
+It is not applicable for [Read Committed](../read-committed/) isolation.
+
+1. In this mode, transactions are assigned random priorities with some exceptions as described in [Transaction Priorities](../transaction-priorities/).
+2. If a conflict occurs, the transaction with lower priority is aborted. There are two possibilities when a transaction T1 tries to read/ write/ lock a row in a mode conflicting with a other concurrent transactions:
+   - **Wound:** If T1 has a priority higher than all the other conflicting transactions, T1 will abort them and make progress.
+   - **Die:** If any other conflicting transaction has an equal or higher priority than T1, T1 will abort itself.
+
+```sql
+create table test (k int primary key, v int);
+insert into test values (1, 1);
+```
+
+### Wound example
+
+<table class="no-alter-colors">
+<thead>
+    <tr>
+    <th>
+    Client 1
+    </th>
+    <th>
+    Client 2
+    </th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+   <td>
+   </td>
+   <td>
+
+```sql
+SET yb_transaction_priority_upper_bound = 0.4;
+```
+
```

Comment:
Done, btw I realized that this happens only for the sql statements that spill into more than 1 line on the UI. Maybe we can fix this in our generic styling for future and so that it works well on small screens too